change/@ni-nimble-components-297ab22e-342a-48cb-94f9-a5d8e7fe9bfa.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@DStavilaNI could you please fill in the PR description and title? It helps reviewers understand what it's about and also provides better repo history.

Reply:
Sorry - the details were there, it's just that I did not notice they were commented out :) ... fixed it.

Reply:
@DStavilaNI please update the PR title
![Screenshot_20221129-071516](https://user-images.githubusercontent.com/1588923/204538793-b4cf532f-cd31-4436-9851-15a6798be31c.png)

Reply:
done